plugin/kubernetes/xfr.go

```diff
@@ -31,7 +31,7 @@ func (k *Kubernetes) Transfer(ctx context.Context, state request.Request) (int,
 
 	// Get all services.
 	rrs := make(chan dns.RR)
-	go k.transfer(rrs, state.Zone)
+	go k.transferAndClose(rrs, state.Zone)
```

Comment:
So, after this merges, can we deprecate the transfer options within k8s, and move them all to transfer? Can we use transfer more than once with different zones so we can control the "allowed" IPs on a per-zone basis?

Reply:
Yes to both